docs/contribute/benchmarks/TRAINING/benchmarks_DL1_DirectionLUT.ipynb

```diff
@@ -0,0 +1,813 @@
+{
```

Comment:
<p>Would be good to put z-axis label on all of these, since I find it confusing to tell what is being plotted, and when it's a unitless or not (same for Energy LUTs).  THis section says "same", but it's not the same plot as the last section, I think, it's the LUT, not the comparison, so a label would help.  You can also use <code>plt.suptitle("overall title of both plots")</code></p><p> </p><pre class="ql-syntax" spellcheck="false">cb = plt.colorbar()
cb.set_label("Miss comparison (protopipe/ctamars)")
</pre>
<br/>

 _Reply via <a href='https://app.reviewnb.com/cta-observatory/protopipe/pull/62/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='35'/>

Reply:
<p>OK I'll add them thanks</p><p> </p><p>regarding the "same" plot, it is indeed the same (the ratio between LUTs) it's just that the scale in the first plot is dominated by that cell where the ratio is orders 10^8 so when we zoom into the quality selection images regime the scale goes only from 0 to 7</p>

Reply:
<p>DONE</p>